2025-03-03 14:34:22,496 - INFO - ‚úÖ Using GPU device 1.
2025-03-03 14:34:22,498 - INFO - üìÅ Results will be saved in: /caefs/user/mmingyeong/structura/Structura/src/results
2025-03-03 14:34:23,176 - ERROR - Unhandled exception occurred:
Traceback (most recent call last):
  File "converter_ex.py", line 2, in <module>
    from structura.SimulationDataConverter import SimulationDataConverter
  File "/caefs/user/mmingyeong/structura/Structura/src/structura/__init__.py", line 19, in <module>
    from .SimulationDataConverter import SimulationDataConverter  # ‚úÖ Ï∂îÍ∞Ä
ModuleNotFoundError: No module named 'structura.SimulationDataConverter'
